pkg/webhook/deploy/deployment.go

```diff
@@ -3,11 +3,19 @@ package deploy
 import (
 	"context"
 	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"testing"
```

Comment:
The test functions should be moved to the test file unless I'm missing something

Reply:
Good point! done in https://github.com/codeready-toolchain/member-operator/pull/562/commits/e07e190754ac98ee8781e514a63c1053cc16e628 thanks!